### ISSUE TITLE
Update messages with release

### ISSUE THREAD

**Issue by kizza** [kizza/Table-of-comments#33]
I have a feeling that package control combines the update messages for people who are upgrading through several versions of the plugin.

ie. If we have message files 1.txt, 2.txt, 3.txt and 4.txt (just using simple numbers here).  If someone upgrades from version 1 to version 4 they will see a window the combines the text from all four files (which makes sense for cumulative updates) - so I suspect we should remove the large ascii heading on update messages going forward so that people don't possibly see it multiple times

Have attached a screenshot of a simple upgrade message for thought - it seems most plugins keep these messages smallish and simply formatted so that they look similar across package control.
![package-updates](https://cloud.githubusercontent.com/assets/1088717/6013766/e92d98a6-aba4-11e4-9d0e-d8aa1c54f1c6.jpg)

Any thoughts?

**Comment by ionutvmi** [comment]
That is true, but if we remove the text it won't show at all for future updates.
Lets say I'm 1.1.6 and I update to 1.1.7

we can assume that in most of the cases the update will be only one step
some users who go in vacation and come back will get multiple update messages but that is fine I guess...

Alternatively we could just remove the ascii text and just keep a plain old text
```
Table of comments - UPDATE
=============================
```

Or use the same format as the Cyanide package does...

**Comment by kizza** [comment]
Just checked out Cyanide, and maybe that's nicer using markdown.  I added a "messages" branch where I just chucked in the install ascii title (thinking it uses the full plugin name which could be easier to recognise).  
![screenshot](https://cloud.githubusercontent.com/assets/1088717/6033457/b4851fda-ac61-11e4-8acd-bc55050a0363.png)

If simpler looks best... then yeah probably going with the .md approach gives consistency and makes it look nice in github.  Happy for more thoughts
